src/main/java/com/nike/cerberus/error/DefaultApiError.java

```diff
@@ -269,6 +269,22 @@
      */
     EXPIRED_AWS_TOKEN(99238, "The security token included in the request is expired.", SC_UNAUTHORIZED),
 
+    /**
+     * AWS token expired.
```

Comment:
This comments are copy pasted and unchanged.

- [ ] Update the comments to reflect what the errors are actually for.